### ISSUE TITLE
Cutadapt problem - not cutting some ASVs

### ISSUE THREAD

**Issue by pdcountway** [benjjneb/dada2#1004]
Hi - We're looking at a subset of 20 ASVs detected in a recent 18S sequencing project - all identified as fish (most likely Alosa pseudoharengus - alewife). I used the cutadapt protocol, exactly as described in the dada2 ITS Pipeline Workflow, prior to calling ASVs with dada2. It appears that cutadapt only worked on both ends for 11 of the 20 ASVs. Primer regions for the 9 remaining ASVs were missed at either the 5' or 3' ends. I've attached screen shots of both ends of the ASVs - aligned to some related fish species (genus indicated) with the Euk18S primers listed under the consensus sequence. It's also concerning that ASV00757, at the top of the alignment, seems to have made it through quality/chimera checking since the first ~100 bp are all Cs and Ts. I was assuming that cutadapt was getting rid of sequences for which the primer regions could not be found - is that correct? 
![Fish_18S_5prime_end](https://user-images.githubusercontent.com/44147199/81212090-56aeed80-8fa2-11ea-968a-a05695a2de54.png)
![Fish_18S_3prime_end](https://user-images.githubusercontent.com/44147199/81212100-5adb0b00-8fa2-11ea-9dec-03422bbb3530.png)
Do you have any suggestions on why we encountered this high error rate with cutadapt for just one taxonomic group? I'm guessing that this problem is widespread in our data set but haven't looked closely at other groups. It seems like this could be solved by issue #785 ?
Thanks!

**Comment by benjjneb** [comment]
> It's also concerning that ASV00757, at the top of the alignment, seems to have made it through quality/chimera checking since the first ~100 bp are all Cs and Ts.

ASV00757 appears to have a long low-complexity region at the start. This isn't filtered out by default dada2 filtering, but you can activate a low-complexity filter. See the `rm.lowcomplex` option in `filterAndTrim`. You can also filter those after ASV inference. See `?seqComplexity` and `?plotComplexity` for the dedicated dada2 functions for calculating and visualizing sequence complexities.

> Do you have any suggestions on why we encountered this high error rate with cutadapt for just one taxonomic group? I'm guessing that this problem is widespread in our data set but haven't looked closely at other groups. It seems like this could be solved by issue #785 ?

Yes that might solve this quite easily. Can you give that a try and see if these undesirable sequence go away? To modify the ITS tutorial to add that flag:

```
for(i in seq_along(fnFs)) {
  system2(cutadapt, args = c(R1.flags, R2.flags, "-n", 2, 
                             "--discard-untrimmed", # new flag
                             "-o", fnFs.cut[i], "-p", fnRs.cut[i], # output files
                             fnFs.filtN[i], fnRs.filtN[i])) # input files
}
```

**Comment by pdcountway** [comment]
> "--discard-untrimmed", # new flag

The new flag did the trick. It removed all the untrimmed sequences, including the first one in the list that also had the long low-complexity region. I did not use the rm.lowcomplex filter. Overall, 99 ASVs were removed from the original collection using "--discard-untrimmed".
